src/corehost/common/pal.unix.cpp

```diff
@@ -112,11 +124,68 @@ bool pal::is_path_rooted(const pal::string_t& path)
     return path.front() == '/';
 }
 
-bool pal::get_default_extensions_directory(string_t* recv)
+bool pal::get_default_breadcrumb_store(string_t* recv)
+{
+    recv->clear();
+    pal::string_t ext;
+    if (pal::getenv(_X("CORE_BREADCRUMBS"), &ext) && pal::realpath(&ext))
+    {
+        // We should have the path in ext.
+        trace::info(_X("Realpath CORE_BREADCRUMBS [%s]"), ext.c_str());
+    }
+
+    if (!pal::directory_exists(ext))
+    {
+        trace::info(_X("Directory core breadcrumbs [%s] was not specified or found"), ext.c_str());
+        ext.clear();
+        append_path(&ext, _X("opt"));
+        append_path(&ext, _X("corebreadcrumbs"));
```

Comment:
In Windows it is called BreadcrumbStore. On Unix this is abbreviated partially as BreadcrumbS ;) On Windows there is no env variable. We are consistent on Unix on the plural for both env and dir.